src/rustllvm/RustWrapper.cpp

```diff
@@ -411,11 +411,16 @@ extern "C" LLVMRustMetadataRef LLVMRustDIBuilderCreateFunction(
     bool isLocalToUnit,
     bool isDefinition,
     unsigned ScopeLine,
-    unsigned Flags,
     bool isOptimized,
     LLVMValueRef Fn,
     LLVMRustMetadataRef TParam,
     LLVMRustMetadataRef Decl) {
+#if LLVM_VERSION_GE(4, 0)
+  auto Flags = DINode::FlagPrototyped;
+#else
+  unsigned Flags = DINode::FlagPrototyped;
```

Comment:
In this case, yes, but I wanted to keep it consistent with the majority case of `FlagZero`, which [I think needs to be different](https://github.com/rust-lang/rust/pull/37857#discussion_r88695409).